docs/integration/voice.md

```diff
@@ -4,27 +4,63 @@ This section describes what is required to add a PLAY command which will be unde
 
-## LLM Conversation Agent
+!!! note
+    Queue behaviour when adding items by Assist will follow the settings in MA SETTINGS>>CORE>>PLAYER QUEUES CONTROLLER.
+    
+## HA Assist
+![easiest label](../assets/label-easiest.png)
+
+The Music Assistant Integration allows the use of custom intents for initiating playback that are kept completely local. As with all HA intents the structure of the request is strict.  
 
 **Installation Requirements**
 
-- You need to be running HA 2024.6.0 or later
 - You need to be running the latest version of the HA Integration
-- You need a voice assistant which uses a LLM AI conversation agent configured (even if you want to just type in the query)
-- Preferably allow the auto-exposure of MA media players to Assist (otherwise you will have to do that manually)
+- You need a voice assistant configured (even if you want to just type in the query) (Examples: [Cloud Pipeline](https://www.home-assistant.io/voice_control/voice_remote_cloud_assistant/) or [Local Pipeline](https://www.home-assistant.io/voice_control/voice_remote_local_assistant/))
+- Add a directory to your Home Assistant `config` directory named `custom_sentences/en`
+- Add the file found [here](https://github.com/music-assistant/hass-music-assistant/blob/main/custom_sentences/en/music_assistant_PlayMediaAssist.yaml), to that directory.
+- Restart HA or navigate to Developer Tools>> YAML>> YAML Configuration reloading and reload CONVERSATION and INTENT SCRIPT
 
 ### Usage
 
-Any reasonable request to play on a device should be understood by the LLM thus there are no mandatory sentences that must be used. 
+All sentences must:
 
-In some cases it is still advantageous to also setup the `MA Specific Conversation Agent` described below as it provides a more specific prompt for the AI model to use and the LLM will be made aware of and can use this additional functionality. For example, in testing the LLM did not return a good response for `play a list of 5 classic 80's rock tracks` if the `MA Specific Conversation Agent` described in the next section was not configured.
+- start with the words `Play` or `Listen to` followed by the item type `artist/track/album/playlist/radio station`
+- for album and track be optionally followed by `by (the) artist` and then the artist's name
+- then be optionally followed by an area name or device name
+
+#### Acceptable variations
+
+There are acceptable variations to some words and inclusion of the word `the` is optional.
+
+#### Examples
+
+```
+Play the artist Pink Floyd in the kitchen
+Listen to album Jagged Little Pill in the study
+Listen to the album Greatest Hits by the artist James Taylor in the kitchen
+Play track New Year's Day in the bedroom
+Play the song A Hard Day's Night by Billy Joel in the bedroom
+Listen to the playlist Classic Rock in the study
+Listen to the radio station BBC Radio 1 in the bedroom
+```
```

Comment:
Maybe also have an example here without the areas or speaker name attached

Reply:
There is one further down.